### PR TITLE
Simplify schedules page

### PR DESCRIPTION
**Pull request by billpalombi**
Reorganized and abbreviates content

### CODE REVIEW

File: docs/3.0rc/automate/add-schedules.mdx [hunk 1]
```diff
@@ -1,28 +1,98 @@
...
-the `Deployment` class, and in 
-[block-based deployment](/3.0rc/deploy/serve-flows/#block-based-deployments) YAML files.
+### Create schedules in the UI
 
-Support for multiple schedules in `flow.serve`, `flow.deploy`, `serve`, and [worker-based deployments](/3.0rc/deploy/control-runs/workers/) 
-with `prefect deploy` will arrive soon.
-</Tip>
+You can add schedules in the **Schedules** section of a **Deployment** page in the UI.
+To add a schedule select the **+ Schedule** button. 
+A modal dialog will open.
```
Comment (discdiver) [inline docs/3.0rc/automate/add-schedules.mdx:28]:
```suggestion
```

File: docs/3.0rc/automate/add-schedules.mdx [hunk 2]
```diff
@@ -1,28 +1,98 @@
...
+**What about RRule?**
+The UI does not support creating RRule schedules. 
+However, the UI will display RRule schedules that you've created through the command line.
+</Note>
+
+The new schedule appears on the **Deployment** page where you created it. 
+New scheduled flow runs are visible in the **Upcoming** tab of the **Deployment** page.
 
-You can add a schedule to a deployment in the Prefect UI, in the CLI through the `prefect deployment schedule` command, or the `prefect.yaml` configuration file.
+To edit an existing schedule, select the **Edit** from the three-dot menu next to a schedule on a **Deployment** page.
```
Comment (discdiver) [inline docs/3.0rc/automate/add-schedules.mdx:40]:
```suggestion
To edit an existing schedule, select **Edit** from the three-dot menu next to a schedule on a **Deployment** page.
```

**Review by discdiver — COMMENTED** [review]
Thank you! Minor nits.

File: docs/3.0rc/automate/add-schedules.mdx (hunk 1 above)
Comment (billpalombi) [reply docs/3.0rc/automate/add-schedules.mdx]: Agreed. Removed

File: docs/3.0rc/automate/add-schedules.mdx [hunk 3]
```diff
@@ -323,8 +283,7 @@ PREFECT_API_SERVICES_SCHEDULER_MAX_SCHEDULED_TIME='100 days, 0:00:00'
 
 See the [Settings docs](/3.0rc/manage/configure-client/) for more information on altering your settings.
```
Comment (discdiver) [inline docs/3.0rc/automate/add-schedules.mdx:284]:
```suggestion
See the [Settings docs](/3.0rc/manage/settings-and-profiles/) for more information on altering your settings.
```

**Review by discdiver — COMMENTED** [review]
One link on the page, not put there by you, needs a replacement. Suggested a new URL.

**Review by discdiver — APPROVED** [review]

**Pull request merged**